### PR TITLE
Set Stroke end-cap style and join style to ROUND for render and export

### PR DESCRIPTION
**Pull request by HanzPetrov**
Respectfully suggest that rendering and export fidelity would be improved by setting end-cap and join styles to ROUND (instead of default BUTT and MITER), particularly for pen widths > 1 mm

### CODE REVIEW

**Comment by SonarSonic** [comment]
Thanks for your work on these, I think these are all good changes!
There was only one issue I noticed, that might need to be changed / implemented in a different way.

File: src/main/java/drawingbot/files/exporters/Graphics2DExporter.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
 public class Graphics2DExporter {
 
     public static void preDraw(ExportTask exportTask, Graphics2D graphics, int width, int height, PlottingTask plottingTask){
-        graphics.translate(exportTask.exportResolution.getScaledOffsetX(), exportTask.exportResolution.getScaledOffsetY());
+        graphics.translate(exportTask.exportResolution.getScaledOffsetX()+0.5, exportTask.exportResolution.getScaledOffsetY()+0.5);
```
Comment (SonarSonic) [inline src/main/java/drawingbot/files/exporters/Graphics2DExporter.java:21]: In some situations like SVG / PDF exporting the _getScaledOffset_ will not be relative to the printing size not the image size, so 0.5 may not be half a pixel here. I think the 0.5 offset would need to added earlier in the chain, either in the PFM itself or in the "Print Resolution" transforms.

**Review by SonarSonic — APPROVED** [review]

**Review by SonarSonic — COMMENTED** [review]
These are good to merge :)

File: src/main/java/drawingbot/files/exporters/Graphics2DExporter.java (hunk 1 above)
Comment (HanzPetrov) [reply src/main/java/drawingbot/files/exporters/Graphics2DExporter.java:21]: My limited testing of PDF and SVG output showed this fix to be working adequately, but I may be missing something. Can you describe cases where the result would not be as expected? (Btw, it seems to me that addressing this issue in the PFM itself would be a little _too_ early in the chain ;)
And sorry - I probably should have put this on a separate branch... (pushed too soon!)

**Comment by HanzPetrov** [comment]
Hi Ollie,
I'll close this PR and open a new one for Stroke style exports alone (since the render style changes made it to v1.1 and the 0.5 pixel offset changes seem not ready for prime time ;)

**Pull request closed without merging**